big_o_exercise/readme.md

```diff
@@ -36,6 +48,8 @@ function logAtMost10(n) {
         console.log(i);
     }
 }
+Time: O(n)
```

Comment:
double check your time complexity here. What's the largest number of operations that will be performed?